python/cudf/formatting.py

```diff
@@ -49,7 +50,7 @@ def format(index, cols, show_headers=True, more_cols=0, more_rows=0,
     # compute column widths
     widths = {}
     for k, vs in cols.items():
-        if isinstance(k, int):
+        if isinstance(k, utils.int_types_tuple):
```

Comment:
Instead of defining this utility tuple you should be able to do `isinstance(k, np.integer)` as the basic Python `int` type I believe will return true as well as numpy integers of arbitrary bitwidth.

Reply:
I am afraid but basic python `int` is not returning `True` if we I modify the if condition to  
`if isinstance(k, np.integer):`

Reply:
```python
import numbers
isinstance(...k, numbers.Integral)
```

Regardless, even if the `numbers` module wasn't around I'd prefer to avoid hiding the classes we allow in the utils module and instead have them listed here explicitly.

Reply:
`numbers.Integral` seems to be a good suggestion.